ls1intum/Artemis
# `Development`: Update node version in setup documentation

### Checklist
#### General



### Motivation and Context


When setting up a local developing environment I noticed that the node.js version mentioned in the setup documentation was outdated. 

### Description

The node LTS version mentioned in the setup documentation has been changed from (>=16.13.0 < 17) to (>=18.10.0 < 19) so it corresponds to the one specified in the package.json file.

### Steps for Testing

Look at the line changed in the setup documentation, check if the node version is now correct.

### Review Progress






#### Code Review
